granary/meetup.py

```diff
@@ -0,0 +1,105 @@
+# coding=utf-8
+"""Meetup.com source class.
+"""
+
+from . import source
+import logging
+from oauth_dropins.webutil import util
+import urllib.parse, urllib.request
+
+API_BASE = 'https://api.meetup.com'
+API_RSVPS = '/%(urlname)s/events/%(event_id)s/rsvps'
+
+class Meetup(source.Source):
+
+    DOMAIN = 'meetup.com'
+    NAME = 'Meetup.com'
+
+    def __init__(self, access_token):
+        self.access_token = access_token
+        pass
+
+    def create(self, obj, preview=False, include_link=source.OMIT_LINK, ignore_formatting=False):
+        return self._create(obj, preview, include_link, ignore_formatting)
+
+    def preview_create(self, obj, preview=True, include_link=source.OMIT_LINK, ignore_formatting=False):
+        pass
+
+    def post_rsvp(self, urlname, event_id, response):
+        url = API_BASE + API_RSVPS % {
+                'urlname': urlname,
+                'event_id': event_id,
+                }
+        params = 'response=%(response)s' % {
```

Comment:
is this overkill? thinking ahead to when we may want to change it in the future, but likely not helpful

Reply:
seems fine to me. you'd actually use obj['content'] instead of the method param, right? go for it!

Reply:
Sorry what do you mean?